Makefile

```
@@ -92,7 +97,7 @@ zip_dependencies: ## Zip relevant files
 
 .build_deps: build_dep_viewer_app build_dep_simplesettings_app build_dep_user_oidc_app build_dep_ionos_theme
 
-build_release: .build_deps add_config_partials zip_dependencies ## Build a release package (build apps/themes, copy configs and package)
+build_release: .build_nextcloud build_deps add_config_partials zip_dependencies ## Build a release package (build apps/themes, copy configs and package)
```

```suggestion
build_release: .build_deps build_nextcloud add_config_partials zip_dependencies ## Build a release package (build apps/themes, copy configs and package)
```

* .build_deps - should have the dot since its a hidden job
* lets move `build_nextcloud` to "the end" in order to have possibility to have pre-build steps like "build-vue-icons" etc